src/core/api/tasklet_api.cpp

```diff
@@ -52,3 +52,10 @@ bool otTaskletsArePending(otInstance *aInstance)
 {
     return aInstance->mIp6.mTaskletScheduler.AreTaskletsPending();
 }
+
+#ifndef _MSC_VER
```

Comment:
Do you have any better idea on how to implement weak functions in MSVS? Appveyor seems to not like `__declspec(selectany)` attribute. @nibanks ?